ktor-utils/common/src/io/ktor/util/StringValues.kt

```diff
@@ -113,8 +113,15 @@ public interface StringValuesBuilder {
     public operator fun get(name: String): String?
     public fun append(name: String, value: String)
     public fun appendAll(stringValues: StringValues)
-    public fun appendMissing(stringValues: StringValues)
     public fun appendAll(name: String, values: Iterable<String>)
+
+    public fun appendAll(params: Map<String, String>): Unit =
+        params.forEach { (key, value) -> append(key, value) }
+
+    public fun appendAllIterable(params: Map<String, Iterable<String>>): Unit =
+        params.forEach { (key, value) -> appendAll(key, value) }
```

Comment:
Let's convert these functions into extension-functions. This should allow us to rename `appendAllIterable` to `appendAll` and apply `@JvmName("appendAllIterable")` annotation to resolve name clash on JVM.

Probably we should also add extension-functions with varargs:
```kotlin
public fun StringValuesBuilder.appendAll(vararg values: Pair<String, String>) {
    values.forEach { (key, value) -> append(key, value) }
}

@JvmName("appendAllIterable")
public fun StringValuesBuilder.appendAll(vararg values: Pair<String, Iterable<String>>) {
    values.forEach { (key, value) -> appendAll(key, value) }
}
```